src/rust/plugin-registry/src/main.rs

```diff
@@ -0,0 +1 @@
+fn main() {}
```

Comment:
This could be a more useful stub, along the lines of what's present for the work queue.